smf/clickhouse/method_script.sh

```diff
@@ -28,13 +28,136 @@ ipadm show-addr "$DATALINK/ll" || ipadm create-addr -t -T addrconf "$DATALINK/ll
 ipadm show-addr "$DATALINK/omicron6"  || ipadm create-addr -t -T static -a "$LISTEN_ADDR" "$DATALINK/omicron6"
 route get -inet6 default -inet6 "$GATEWAY" || route add -inet6 default -inet6 "$GATEWAY"
 
-args=(
+# TEMPORARY: Racks will be set up with single node ClickHouse until
+# Nexus provisions services so there is no divergence between racks
+# https://github.com/oxidecomputer/omicron/issues/732
+single_node=true
```

Comment:
How will this be changed? We'll make a commit which changes it?

At that point, when we boot this new zone...what happens? There already is a database `oximeter`. Now, we can delete that ourselves manually, by just removing the files. But if we don't do that, we'll then run `CREATE DATABASE oximeter ON CLUSTER oximeter_cluster`. Does that conflict? When we create those tables, do those conflict?

Reply:
I've been mulling it over, but tbh I can't say I have a clear answer as to which steps we'll take for the migration (or perhaps remove everything and start from scratch?). I was planning on having that discussion here -> https://github.com/oxidecomputer/omicron/issues/4000 I think a lot of it also depends on "how" Nexus will provision services.

If we are able to perform a one off job for the migration, we may be able to use [`remote()`](https://clickhouse.com/docs/en/sql-reference/table-functions/remote) after renaming the old tables or something like that.